### PR TITLE
Add service mode in faq.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -82,3 +82,18 @@ You can find the details about how do we calculate these metrics [here](https://
 
 In the overview blade of Azure SignalR Service resources, we have already chosen the approperate aggregation type for you. And if you go to the Metrics blade, you can
 take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsignalrservicesignalr) as a reference.
+
+<a name="service-mode"></a>
+## What is the meaning of service mode `Default`/`Serverless`/`Classic`? how can I choose?
```
Comment (chenkennt) [inline docs/faq.md]: how -> How

File: docs/faq.md [hunk 2]
```diff
@@ -82,3 +82,18 @@ You can find the details about how do we calculate these metrics [here](https://
 
 In the overview blade of Azure SignalR Service resources, we have already chosen the approperate aggregation type for you. And if you go to the Metrics blade, you can
 take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsignalrservicesignalr) as a reference.
+
+<a name="service-mode"></a>
+## What is the meaning of service mode `Default`/`Serverless`/`Classic`? How can I choose?
+
+Modes:
+* `Default` mode **require** hub server. When a hub without any available server connection, client connection will be reject.
```
Comment (vicancy) [inline docs/faq.md]: >require [](start = 19, length = 7)

`require` => `requires`
`When there is no server connection available for the hub, the client tries to connect to this hub fails.`?

File: docs/faq.md [hunk 3]
```diff
@@ -82,3 +82,18 @@ You can find the details about how do we calculate these metrics [here](https://
 
 In the overview blade of Azure SignalR Service resources, we have already chosen the approperate aggregation type for you. And if you go to the Metrics blade, you can
 take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsignalrservicesignalr) as a reference.
+
+<a name="service-mode"></a>
+## What is the meaning of service mode `Default`/`Serverless`/`Classic`? How can I choose?
+
+Modes:
+* `Default` mode **require** hub server. When a hub without any available server connection, client connection will be reject.
+* `Serverless` mode do **NOT** allow any server connection, i.e. it will reject all server connections, all clients must in serverless mode. This mode allow dynamic hub.
```
Comment (vicancy) [inline docs/faq.md]: >do [](start = 20, length = 2)

`do` => `does`

File: docs/faq.md [hunk 4]
```diff
@@ -82,3 +82,18 @@ You can find the details about how do we calculate these metrics [here](https://
...
 In the overview blade of Azure SignalR Service resources, we have already chosen the approperate aggregation type for you. And if you go to the Metrics blade, you can
 take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsignalrservicesignalr) as a reference.
+
+<a name="service-mode"></a>
+## What is the meaning of service mode `Default`/`Serverless`/`Classic`? How can I choose?
+
+Modes:
+* `Default` mode **require** hub server. When a hub without any available server connection, client connection will be reject.
+* `Serverless` mode do **NOT** allow any server connection, i.e. it will reject all server connections, all clients must in serverless mode. This mode allow dynamic hub.
+* `Classic` mode is a mixed status. When a hub has server connection, the new client will be route to hub server, if not, client will enter serverless mode.
```
Comment (vicancy) [inline docs/faq.md]: >route [](start = 93, length = 5)

`route` => `routed`

**Comment by sffamily** [comment]
#sign-off

**Pull request merged**